sas-9.4m7-VA-MPP/azuredeploy.parameters.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
remove the parameters for which you have defaultValues so they can be tested

Reply:
In order to even start the deployment in the CI environment, I had to provide non-default values for these parameters:
location: "eastus"
    - otherwise the job runs in westus, which doesn't support availability zones
various VM sizes: "Standard_B2ms"
Visual Analytics Worker Count: 1
   - otherwise the number of cores will exceed the CI environment limit of 10

Reply:
ok, re: location - you can use westus2 - that has quota for DsV3 series and zones.

re: the defaultValue for location in the parameter - remove the defaultValue property since that may block deployment if the RG is in a location that doesn't support zones.